tests/profiler/tensorflow2/test_native_tf2_profiler.py

```diff
@@ -146,7 +146,10 @@ def _verify_timeline_files(out_dir):
             "process_sort_index",
         }
 
-
+# Skipping because the tests are failing.
+# Support for profiling using gradient tape has never been released publicly
+# and since we're planning on deprecating profiler v1, we can just disable the tests
+@pytest.mark.skip(reason="unblock TF2.7 release")
```

Comment:
The tests only fail for TF 2.7, right? Then let's use `skipif` to skip the test for all TF versions 2.7 and above.

Reply:
Hi @ndodda-amazon, do tests under `tests/profiler/tensorflow2/` depends on the code changes applied to the tf code base?